tests/ImageSharp.Tests/IO/BufferedReadStreamTests.cs

```diff
@@ -235,34 +215,28 @@ public void BufferedStreamCanReadSubsequentMultipleByteSpanCorrectly(int bufferS
     public void BufferedStreamCanSkip(int bufferSize)
     {
         this.configuration.StreamProcessingBufferSize = bufferSize;
-        using (MemoryStream stream = this.CreateTestStream(bufferSize * 4))
-        {
-            byte[] expected = stream.ToArray();
-            using (var reader = new BufferedReadStream(this.configuration, stream))
-            {
-                int skip = 1;
-                int plusOne = 1;
-                int skip2 = bufferSize;
+        using MemoryStream stream = CreateTestStream(bufferSize * 4);
+        byte[] expected = stream.ToArray();
+        using BufferedReadStream reader = new(this.configuration, stream);
+        const int skip = 1;
+        const int plusOne = 1;
+        int skip2 = bufferSize;
 
-                // Skip
-                reader.Skip(skip);
-                Assert.Equal(skip, reader.Position);
-                Assert.Equal(stream.Position, reader.Position);
+        // Skip
+        reader.Skip(skip);
+        Assert.Equal(skip, reader.Position);
```

Comment:
Tests were bad here. It was wrong to expect the underlying position to match since we often go out of sync by filling the buffer.